src/background/chrome-api.js

```diff
@@ -36,9 +45,13 @@ export function getChromeAPI(chrome = globalThis.chrome) {
    * @param {(...args: [...Args, Callback<Result>]) => void} fn
    * @return {(...args: Args) => Promise<Result>}
    */
-  const promisify =
-    fn =>
-    (...args) => {
+  const promisify = fn => {
+    const cached = cache.get(fn);
+    if (cached) {
+      return cached;
+    }
+
+    return (...args) => {
```

Comment:
This resolves a comment I left in #663 ... the explicit `return` here is easier to read. Thanks!